lib/load-logos.js

```diff
@@ -12,7 +12,8 @@ var loadLogos = function() {
 
     // eg, github
     var name = filename.slice(0, -('.svg'.length));
-    logos[name] = 'data:image/svg+xml;base64,' + Buffer(svg).toString('base64');
+    logos[name] = 'data:image/svg+xml;base64,' +
+      (new Buffer(svg)).toString('base64');
```

Comment:
Could we use `Buffer.from` instead? It's been around [since at least Node 4](https://nodejs.org/dist/latest-v4.x/docs/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe). `new Buffer` was [marked deprecated](https://nodejs.org/dist/latest-v6.x/docs/api/buffer.html) in Node 6.